README.rst

```diff
@@ -50,7 +50,8 @@ DNS lookup...
   the lookup.
 * If ``gethostbyname`` does not have it cached then a request is made to the
   known DNS server that was given to the network stack. This is typically the
-  local router or the ISP's caching DNS server
+  local router or the ISP's caching DNS server.
+* Port 53 is opened to send UDP request to DNS server.
```

Comment:
I check http://serverfault.com/questions/181956/is-it-true-that-a-nameserver-have-to-answer-queries-over-tcp and RFC and indeed browser request may use TCP.